contrib/node/src/python/pants/contrib/node/targets/node_module.py

```diff
@@ -6,18 +6,32 @@
                         unicode_literals, with_statement)
 
 from pants.base.payload import Payload
+from pants.base.payload_field import PrimitiveField
 
 from pants.contrib.node.targets.node_package import NodePackage
 
 
 class NodeModule(NodePackage):
   """A Node module."""
 
-  def __init__(self, sources=None, address=None, payload=None, **kwargs):
+  def __init__(
+    self, sources=None, build_script=None, output_dir='dist',
+    dev_dependency=False, address=None, payload=None, **kwargs):
     """
     :param sources: Javascript and other source code files that make up this module; paths are
                     relative to the BUILD file's directory.
     :type sources: `globs`, `rglobs` or a list of strings
+
```

Comment:
These make a lot more sense now, thanks.

One last bit of confusion is the `dev_dependencies` argument. Is that marking a whole target as a dev_dependency? I guess that would be because the `node_tests` target doesn't actually own any sources of its own, and thus needs to get them from some other target that has the dev dependency?

I guess that's not really a thing we can fix unless we're able to encourage people to split their target into "target" and "test target", as most other languages do.